src/evm/mod.rs

```diff
@@ -178,11 +179,14 @@ pub fn evm_xcontract_call_using_batch(
 ) -> Result<bool, ethabi::Error> {
     use std::convert::TryFrom;
     let rt_env = RuntimeEnvironment::new(Uint256::from_usize(1111));
+
+    let wallet = rt_env.new_wallet();
+    let addr_h160: Address = wallet.private_key().into();
+    let my_addr = Uint256::from_bytes(addr_h160.as_bytes());
```

Comment:
Is wallet of type `ethers_signers::Wallet`? If so it seems like we should just use `wallet.address()` here

Reply:
Switched to using `wallet.address()`.  But I still need to convert it to a Uint256.